static/include/robot/apis/generated/robot.md

```diff
@@ -0,0 +1,760 @@
+await machine.stopAll();
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Log
+
+Create a LogEntry object from the log to send to the RDK over gRPC.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `name` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The logger’s name.
+- `level` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The level of the log.
+- `time` ([datetime.datetime](https://docs.python.org/3/library/datetime.html)) (required): The log creation time.
+- `log` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The log message.
+- `stack` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The stack information of the log.
+
+**Returns:**
+
+- None.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.log).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetCloudMetadata
+
+Get app-related information about the robot.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- ([viam.proto.robot.GetCloudMetadataResponse](https://python.viam.dev/autoapi/viam/proto/robot/index.html#viam.proto.robot.GetCloudMetadataResponse)): App-related metadata.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.get_cloud_metadata).
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context#Context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+**Returns:**
+
+- [(cloud.Metadata)](https://pkg.go.dev/go.viam.com/rdk/cloud#Metadata)
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+**Example:**
+
+```go {class="line-numbers linkable-line-numbers"}
+metadata := machine.GetCloudMetadata()
+machine_part_id = metadata.MachinePartID
+primary_org_id = metadata.PrimaryOrgID
+location_id = metadata.LocationID
+```
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/robot#Robot).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Options.with_api_key
+
+Create a `RobotClient.Options` using an API key as credentials.
+Pass these options to [`AtAddress`](#ataddress).
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `api_key` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): your API key
+- `api_key_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): your API key ID. Must be a valid UUID
+
+**Returns:**
+
+- ([typing_extensions.Self](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient)): the RobotClient.Options
```

Comment:
This is a little odd. Wonder if we should replace with `viam.robot.client.RobotClient` or just `RobotClient`

Reply:
Guess: `typing_extensions.Self` is relative to calling function, could be more than just `RobotClient` elsewhere. For now, I want to match to only case I've found (for `robot` resource), but expecting to have to walk (some of) this guess back later. For now, this is my best balance: return data type will always be correct (as-scraped), while possibility of incorrect link target exists.

If I complete all autogen merges and this return data type is only used for `RobotClient`, I will indeed replace the data type name everywhere.